lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
@nmote 
We have two options for this:
1. Add setting for disable
2. Try to start flow (replace which flow to just start flow) - is not good for performance reasons (because it will try to start flow twice, and for each file).

I've also noticed that for some another extensions global node is also required and also for Debug. So this check can be a feature that can explain user why Debug is not work.